charts/cluster-proportional-autoscaler/templates/serviceaccount.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.serviceAccount.create -}}
+{{- if and .Values.rbac.create .Values.serviceAccount.create -}}
```

Comment:
This is used in the deployment and should be created even if there are no roles to be bound to it